src/axolotl/utils/callbacks/__init__.py

```
@@ -66,6 +65,14 @@ def on_step_end(
         control: TrainerControl,
         **kwargs,
     ) -> TrainerControl:
+        # Import error due Transformers removed tensorflow support after version 4.56.2.
+        # Need wait until optimum release a new version.
+        try:
+            from optimum.bettertransformer import BetterTransformer
+        except ImportError as e:
+            raise ImportError(
+                f"Import error due Transformers removed tensorflow support after version 4.56.2. Please install transformers<=4.56.2: {e}"
+            )
```

Ruff (0.13.1)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)